IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -81,6 +82,7 @@ - (nullable instancetype)initWithRequestParameters:(nonnull MSIDRequestParameter
         _tokenResponseValidator = tokenResponseValidator;
         _tokenResponseHandler = [MSIDTokenResponseHandler new];
         _lastRequestTelemetry = [MSIDLastRequestTelemetry sharedInstance];
+        _unexpiredRefreshNeededAccessToken = NULL;
```

Comment:
no need to initialize with NULL. All Objective-C properties would be `nil` by default.

Reply:
Done